internal/server/v0/triple/golden/get_triples/stat_var.json

```diff
@@ -556,34 +556,6 @@
       ],
       "provenanceId": "dc/d7tbsb1"
     },
-    {
-      "subjectId": "Percent_AtleastTwiceADay_ConsumptionOfFruits_In_Count_Person",
```

Comment:
suspicious deletion

Reply:
https://screenshot.googleplex.com/53JkaRvFskSyHBB.

Percent_AtLeastTwiceADay_ConsumptionOfVegetables_In_Count_Person is there above.